### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by shiffman** [shiffman/Neural-Network-p5#7]
Right now it calculates a percentage completed since the sketch first begins. There's probably a better way to think about this.

**Comment by lazoviccorp** [comment]
The traditional way is to query all of testing data at once and calculate the accuracy, and do this every so often, lets say every 100 iterations...

**Comment by shiffman** [comment]
Indeed, one reason why I didn't do this is that I would like the animation to run continuously at a relatively fast framerate. Running through all the data cannot be done reasonably on a per frame basis. However, I could only update the performance rating once it's made a full pass over the test data (and possibly increase to 2, 3, etc. data points per frame.)
